libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -15,7 +20,10 @@ import {
 })
 export class FieldsService {
   private fields = {
-    publisher: new SimpleSearchField('OrgForResource', 'asc', this.injector),
+    publisher:
```

Comment:
I wonder if it wouldn't be better to have a `OrganizationSearchField` which would handle that, keeping the root service lighter. It's a bit intrusive here, the `if` statement and the injection.

Reply:
I don't know, both fields have different implementations so the `OrganizationSearchField` would contain a lot of if's...